tests/test_flow_diabetes.py

```
@@ -11,15 +11,16 @@
     # Note: We're using a mock implementation for AI summarizer that doesn't require OpenAI
 
     # Create user
-    u = User.objects.create_user(username='doc1', password='p1')
+    User = get_user_model()
```

Ruff (0.12.2)

Variable `User` in function should be lowercase

(N806)